app/lib/discourse_automation/scripts/send_pms.rb

```diff
@@ -12,7 +12,7 @@
   field :receiver, component: :user, triggerable: :recurring
   field :sendable_pms, component: :pms, accepts_placeholders: true
 
-  triggerables %i[user_added_to_group stalled_wiki recurring]
+  triggerables %i[user_added_to_group stalled_wiki recurring user_promoted]
```

Comment:
Not sure if we actually want this as a triggerable for the send PM script; just needed it at least on one thing for testing.